models/SupporterIndividual.js

```diff
@@ -8,12 +8,35 @@ var Types = keystone.Field.Types;
 
 var SupporterIndividual = new keystone.List('SupporterIndividual', {
 	map: { name: 'name' },
-	autokey: { path: 'slug', from: 'name', unique: true }
+	autokey: { from: 'name', path: 'slug', unique: true },
+	sortable: true
 });
 
+var dateParseFormat = 'DD/MM/YYYY';
+
 SupporterIndividual.add({
-	name: { type: Types.Name, required: true },
-});
+	name: { type: String, required: true },
+	key: { type: Types.Key, noedit: true },
+	email: { type: Types.Email },
+	image: { type: Types.CloudinaryImage },
+	message: { type: Types.Markdown},
+	donationAmount: {type:Number},
+	paymentCompleted: { type: Boolean},
```

Comment:
There might be some confusion around donations and supporters here.
We might need some advice from Nicola here @ourschoolliberia 

`SupporterIndividuals` and `SupporterCompanies` were created to be an admin managed list. In my understanding reserved for heavier supporters rather than random donations, whereas a Donation object is created to represent an individual donating to the site, and triggered an email to admins.

In this new implementation, anyone who donates, whether they wish to be anonymous or not would have a `SupporterIndividual` object created and on the Supporters screen an extra item would appear. I assume the `published` boolean is there for admin oversight? That should definitely be set false until admin decides incase someone puts in "Massive Cock" as their name etc.. Also there's no longer emails pushed to admin, unless paypal takes care of that?

Reply:
Correct, the idea was to treat the list of supporter sort of as a task list for the administrator. This would allow to contact and then either publish or delete the entry.